scripts/zones/AlTaieu/mobs/Jailer_of_Prudence.lua

```diff
@@ -78,26 +135,17 @@ entity.onMobDeath = function(mob, player, optParams)
         xi.roe.onRecordTrigger(player, 770)
         player:setLocalVar('prudenceCount', 0)
     end
-end
 
-entity.onMobDespawn = function(mob)
-    if mob:getID() == ID.mob.JAILER_OF_PRUDENCE then
-        local secondPrudence = GetMobByID(ID.mob.JAILER_OF_PRUDENCE + 1)
+    if optParams.isKiller then
```

Comment:
`if optParams.isKiller or optParams.noKiller then`

`optParams.noKiller` is used when the mob dies from a DoT effect, for instance.
This is me assuming you want this logic executing once, but guaranteing it actually executes.

Reply:
Got that adjusted